### DISCUSSION TITLE
creating characters programatically

### DISCUSSION THREAD

**Discussion by ajadavis** [pixiv/vroid-sdk-developers#56] [Q&A]
is this possible? i would love to deploy a server instance that takes in an api call and returns a new character. the sdk looks to be involved with unity and i am not too experienced with unity though i could figure it out with some time. just looking for some general advice and guidance from more experienced folks if this will end up being possible for me to achieve.

**Comment by nyamadan** [comment]
VRoid SDK can load 3D models from VRoid Hub, and upload 3D models to VRoid Hub.
However, it cannot generate 3D models.

UniVRM may be able to generate 3D models that can be uploaded to VRoid Hub.
https://github.com/vrm-c/UniVRM#export-features
